### DISCUSSION TITLE
Add model/texture name/keyword search function

### DISCUSSION THREAD

**Discussion by shrine-engie** [Garux/netradiant-custom#179] [Ideas]
Please and thank you.

**Comment by Garux** [comment]
![image](https://github.com/Garux/netradiant-custom/assets/1734117/8ae32afd-b6bf-4b25-a446-dc8299f6bd5f)
![image](https://github.com/Garux/netradiant-custom/assets/1734117/74ff5dc8-933a-4758-acf1-5efc7395cf59)
![image](https://github.com/Garux/netradiant-custom/assets/1734117/c344da36-eccf-4a24-9270-8e6bff0cd52c)

**Reply by shrine-engie** [reply]
I was thinking more along these lines. A universal search bar that lets users work within the program and see what they're selecting. Any model or texture in the folders NetRadiant scans would show upon searching, including those stored within pk3s. 

![model_search_mockup](https://github.com/Garux/netradiant-custom/assets/155573033/55bab9f2-88e8-4c76-b14f-a7128e1c6b46)
![texture_search_mockup](https://github.com/Garux/netradiant-custom/assets/155573033/aca42380-dc43-45dd-9824-8cc64a50e1ea)


Much of what I work with is stored in pk3s, and I need to use an external program if I want to do broad searches for the files and content within those.

**Comment by Garux** [comment]
These would be nice to have.
However loading is not feee of charge. Too broad search request may take quite some time to load or even hit VRAM capacity available.
